governance/votes/Executive vote - November 28, 2024.md

```diff
@@ -0,0 +1,267 @@
+
+### GSM Pause Delay Increase
+
+- **Authorization**: [Governance Facilitator Approval](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579/2)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579)
+
+If this executive proposal passes, then the [GSM Pause Delay](https://sky-atlas.powerhouse.io/A.1.8.2.1_Pause_Delay/a98b8227-95f6-4711-9d8d-f52cbc6ad2d0%7C0db30758e055) will be increased by 14 hours from 16 hours to **30 hours**.
+
+
+
+### Addition of ilks to the `LineMom`
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594/4)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594/2)
+
+If this executive proposal passes, then the following ilks will be added to the [`LineMom`](https://github.com/makerdao/line-mom/tree/master), allowing the [Standby Spells](https://github.com/makerdao/dss-emergency-spells) to affect them:
+
+- [ALLOCATOR-SPARK-A](https://makerburn.com/#/collateral/ALLOCATOR-SPARK-A)
+- [RWA001-A](https://makerburn.com/#/collateral/RWA001-A)
+- [RWA002-A](https://makerburn.com/#/collateral/RWA002-A)
+- [RWA009-A](https://makerburn.com/#/collateral/RWA009-A)
+- [RWA012-A](https://makerburn.com/#/collateral/RWA012-A)
+- [RWA013-A](https://makerburn.com/#/collateral/RWA013-A)
+- [RWA015-A](https://makerburn.com/#/collateral/RWA015-A)
+
+### Addition of Standby Spells to the Chainlog
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594/4)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594)
+
+[Standby Spells](https://github.com/makerdao/dss-emergency-spells) are reusable pre-deployed spells that bypass the [GSM Pause Delay](https://sky-atlas.powerhouse.io/A.1.8.2.1_Pause_Delay/a98b8227-95f6-4711-9d8d-f52cbc6ad2d0%7C0db30758e055) when executed, so that their effects are immediate.
+
+Standby Spells are formally defined in the Atlas under [article_number]($link_to_atlas_article). Additionally, you can read an introduction to them on [this November 25, 2024 forum post](https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594) by a Governance Facilitator. For detailed technical information, please visit the [GitHub repository at GitHub - makerdao/dss-emergency-spells: Emergency Spells for MakerDAO](https://github.com/makerdao/dss-emergency-spells).
+
+Each Standby Spell is capable of some single action. Standby Spells that trigger some action on _all_ ilks susceptible to having the action applied to them are called _Multi-Ilk Standby Spells_; Standby Spells that trigger some action on _some single_ ilk are called _Single-Ilk Standby Spells_.
```

Comment:
```suggestion
Each Standby Spell is capable of some single action. Standby Spells that trigger some action on _all_ ilks susceptible to having the action applied to them are called _Global/Multi-Ilk Standby Spells_; Standby Spells that trigger some action on _some single_ ilk are called _Single-Ilk Standby Spells_.
```

Reply:
The new official name is Global, but I was asked to use both names on this executive to avoid confusing people